doc/manual/packages.rst

```diff
@@ -570,11 +570,21 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia version to include.
+When Julia version ``0.x`` is released, use ``julia 0.x`` in your ``REQUIRE``. You may consider using ``julia 0.y-`` at this
```

Comment:
No, the only reason to add a requirement on a not-yet-released version is if your package is actually incompatible with the existing releases. I think much of the confusion stems from an intuition-based reading of `0.3-` as "0.3 or newer," without realizing that the first item is a lower bound by default, and upper bounds are a separate second item if they're listed at all.